openmetadata-ui/src/main/resources/ui/src/components/Auth/AppAuthenticators/OidcAuthenticator.tsx

```diff
@@ -91,9 +84,18 @@ const OidcAuthenticator = forwardRef<AuthenticatorRef, Props>(
       setIsSigningUp(true);
     };
 
-    const logout = () => {
-      userManager.removeUser();
-      onLogoutSuccess();
+    const logout = async () => {
+      try {
+        await userManager.signoutRedirect({
+          post_logout_redirect_uri: window.location.origin,
+        });
+      } catch (error) {
```

Comment:
if error is not in use, we can remove, else it will increase sonarCloud warrning.